catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/federation/impl/SortedFederationStrategy.java

```diff
@@ -233,6 +234,8 @@ public QueryResponse federate(List<Source> sources, QueryRequest queryRequest) {
       queryResponse = queryResponseQueue;
       LOGGER.debug("returning returnResults: {}", queryResponse);
     }
+    Map<String, Serializable> props = queryResponse.getProperties();
+    props.put("qm.sort.elapsed", endTime - startTime);
```

Comment:
Isn't this just the time it takes to submit a task to the executor service? That is, it's not the time to complete the task. Was that the intent?

Reply:
The intention is to capture the time that the query takes to sort the results. I suppose a more accurate place for this would be in `SortedQueryMonitor`. Will make that change